scripts/glue-files.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+# SPDX-License-Identifier: ISC
+# Copyright (c) 2018, Ulf Magnusson
+
+# Merge list of given files
+#
+# Usage:
+#
+#   $ python glue-files.py destination src1 src2 [src3 ...]
+
+import shutil
+import sys
+import os
+
+def glue_files(dest, src_list):
```

Comment:
wont ```cat src1 src2 src3 > dest``` do the same thing ?

Reply:
On Windows platform it won't

Reply:
Windows does support running "type" for each file as individual commands and appending with >> so you can do that instead.

Reply:
chnaged